code/modules/vore/eating/living_vr.dm

```diff
@@ -25,7 +25,7 @@
 		return 1
 	M.verbs += /mob/living/proc/insidePanel
 
-	M.appearance_flags |= PIXEL_SCALE
+	//M.appearance_flags |= PIXEL_SCALE
```

Comment:
I think this is beyond the scope of this PR.  If you want to disable pixel scale on all mobs, that needs to be its own discussion, its bigger than dogborgs.